contentcuration/contentcuration/static/js/edit_channel/new_channel/hbtemplates/channel_create.handlebars

```diff
@@ -47,11 +47,14 @@
 		</ul>
 	</div>-->
 	<div id="channel-list-area">
-		<h3>My Channels</h3>
+		<h3>
```

Comment:
https://trello.com/c/RIdTq7zh/222-consider-moving-new-channel-functionality-to-top-of-list